docs/Researcher/best-practices/personal-applications.md

```diff
@@ -0,0 +1,49 @@
+# Personal Applications
```

Comment:
I'm not sure we would like to promote it in this name - Maybe User Applications? not sure